### PR TITLE
job challange completed harshdigwani

### PR DESCRIPTION
**Pull request by harshdigwani**
Hello developers

Hope you all good, kindly accept my pull request.

Thank You

### CODE REVIEW

**Comment by gkanishk** [comment]
Please resolve conflicts and revert changes inside files other than index.md

File: content/participants/ttjc@harshdigwani/index.md [hunk 1]
```diff
@@ -32,12 +46,109 @@ It would be great if someone help me out in improving my backend skills.
...
+
+_hosted link_ : https://harshdigwani.github.io/ <br/>
+_Old Version_   https://harshdigwani.github.io/public/v1/index.html<br/>
+_github link_ : https://github.com/harshdigwani/harshdigwani.github.io
+
+
+
+### Gokart: An Ecommerce WebApp
 
 _description_ Gokart an E-Commerce application which solves the problem of finding the best deals on electronic products near you using user's location.
```
Comment (gkanishk) [inline content/participants/ttjc@harshdigwani/index.md:154]: Please remove old projects, after that you can raise **PR** for the finalist

Comment (harshdigwani) [reply content/participants/ttjc@harshdigwani/index.md]: Done sir!

**Comment by gkanishk** [comment]
Add finalist file also, can refer #935 for details

**Pull request merged**